pkg/rhel/openvdc.spec

```diff
@@ -6,7 +6,7 @@
 Name: openvdc
 Version: 0.9%{?dev_release_suffix:dev}
 Release: %{release}%{?dist}
-Summary: Metapackage that depends on all other OpenVNet packages.
+Summary: Metapackage that depends on all other OpenVDc packages.
```

Comment:
Should be OpenVDC with capital C. Not gonna stop the PR for that though. Just make sure to fix it later.